src/Model/ArrayData.php

```diff
@@ -16,14 +17,9 @@
  * ));
  * </code>
  */
-class ArrayData extends ModelData
+class ArrayData extends ModelData implements JsonSerializable
```

Comment:
This was needed for a test which serialises data as a way to quickly check types, or something along those lines. Since arrays get cast to `ArrayData` in some cases, this was needed.

It's also a bit of a no-brainer, since `ArrayData` represents an associative array, you should be able to pass it into `json_encode()`.